src/Explorer/Tabs/TerminalTab.tsx

```diff
@@ -42,16 +44,22 @@ export default class TerminalTab extends TabsBase {
   public readonly html = '<div style="height: 100%" data-bind="react:notebookTerminalComponentAdapter"></div>  ';
   private container: Explorer;
   private notebookTerminalComponentAdapter: NotebookTerminalComponentAdapter;
+  private index: number;
```

Comment:
>private index: number; [](http://example.com/codeflow?start=2&length=22)

should/can this be in TabsBase so all tabs will get it?

what about cassandra shell?

Reply:
done